newIDE/app/src/Utils/GDevelopServices/Usage.js

```diff
@@ -190,6 +190,29 @@ export const businessPlan: PlanDetails = {
   descriptionBullets: [],
 };
 
+const SUBSCRIBED_SIZE_LIMIT_IN_MB = 250;
+const GUEST_SIZE_LIMIT_IN_MB = 50;
+
+export type ExportSizeOptions = {|
+  limit: number,
+  getErrorMessage: (fileSizeInMb: number) => string,
+|};
+
+export const onlineWebExportSizeOptions: {
+  [key: 'guest' | 'subscribed']: ExportSizeOptions,
+} = {
+  guest: {
+    limit: SUBSCRIBED_SIZE_LIMIT_IN_MB * 1000 * 1000,
+    getErrorMessage: (fileSizeInMb: number) =>
+      `Archive is of size ${fileSizeInMb} MB, which is above the limit allowed of ${SUBSCRIBED_SIZE_LIMIT_IN_MB} MB`,
```

Comment:
Should it be `... above the allowed limit of ...`?

Reply:
I still feel that a use seeing this will think "Well great, I have a subscription and I have a limit, and above all I have no idea how to solve this problem"